tests/unittest/llmapi/test_llm_multi_gpu.py

```
@@ -455,6 +455,7 @@ def test_llm_get_stats_async_tp2(pytorch_backend):
     llm_get_stats_async_test_harness(tp_size=2, pytorch_backend=pytorch_backend)
 
 
+@skip_ray
```

Ruff (0.14.8)

Undefined name `skip_ray`

(F821)